datalad/interface/common_opts.py

```diff
@@ -33,6 +33,13 @@
     constraints=EnsureInt() | EnsureNone(),
     doc="""limit recursion into subdataset to the given number of levels""")
 
+shared_access_opt = Parameter(
+    args=('--shared-access',),
```

Comment:
I don't see how command name choice directly relates to the option name here in question, but ok, if you see them needing to be differentiated, I can silence my whining for a bit longer ;-)